modules/@apostrophecms/schema/ui/apos/components/AposInputSelect.vue

```diff
@@ -43,24 +43,6 @@ export default {
       return [ this.value.duplicate && 'apos-input--error' ];
     }
   },
-  async mounted() {
-    // Add an null option if there isn't one already
-    if (!this.field.required && !this.choices.find(choice => {
-      return choice.value === null;
-    })) {
-      this.choices.unshift({
-        label: '',
-        value: null
-      });
-    }
-    this.$nextTick(() => {
-      // this has to happen on nextTick to avoid emitting before schemaReady is
-      // set in AposSchema
-      if (this.field.required && (this.next == null) && (this.choices[0] != null)) {
-        this.next = this.choices[0].value;
-      }
-    });
-  },
```

Comment:
moved to `AposInputChoicesMixin.js` in order to wait for async stuff to happen before doing that